src/logic/operate.js

```diff
@@ -1,17 +1,39 @@
 import Big from 'big.js';
 
+const OPERATORS = ['+', '*', '/', '-'];
+
 const operate = (numberOne, numberTwo, operation) => {
-  const num1 = new Big(numberOne);
-  const num2 = new Big(numberTwo);
+  const calculate = (num1, num2, op) => {
+    switch (op) {
+      case '+': return num1.plus(num2).toFixed();
+      case '*': return num1.times(num2).toFixed();
+      case '/':
+        if (parseInt(num2, 10) === 0) {
+          return '0';
```

Comment:
should return  `Division by zero error` not `0` because displaying `0` those not means division error